SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -83,3 +113,128 @@ FReply FSpatialGDKEditorLayoutDetails::ClickedOnButton()
 
 	return FReply::Handled();
 }
+
+bool FSpatialGDKEditorLayoutDetails::TryConstructMobileCommandLineArgumentsFile(FString& CommandLineArgsFile)
+{
+	UE_LOG(LogSpatialGDKEditorLayoutDetails, Warning, TEXT("Pushing to iOS device..."));
```

Comment:
Is this a Warning or Info log? It looks like Info type.

Reply:
ah, think that can be removed. that was for early testing :)